Jenkinsfile

```diff
@@ -50,7 +50,23 @@ pipeline {
               environment {
                 MSBUILDDEBUGPATH = "${env.WORKSPACE}"
               }
+              /**
+              Make sure there are no code style violation in the repo.
+              */
               stages{
+                stage('CodeStyleCheck') {
+                  steps {
+                    withGithubNotify(context: 'CodeStyle check', tab: 'tests') {
```

Comment:
the tab should be the default one, simply remove the `tab` field, this does not generate a xunit file with tests results

Reply:
Thanks, done!